catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/NsiliEndpoint.java

```diff
@@ -210,6 +212,13 @@ public void setOutgoingValidationEnabled(boolean outgoingValidationEnabled) {
         }
     }
 
+    public void setEmailSender(EmailSender emailSender) {
```

Comment:
I would add javadoc indicating that emailSender shouldn't be null and add a 'notNull' check in the code.